packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -10,22 +10,28 @@ import {
 
 const root = process.cwd();
 
-// Determine root of project to build
+// Determine which build runner to use
+let cmd = process.env.MONOREPO_COMMAND || DEFAULT_COMMAND;
+
+// Read environment variable (only relevant for monorepos with multiple targets)
```

Comment:
Could you note what target means in this case in the comment too, I think it seems to indicate if the project is a monorepo or not, but that's not entirely clear

Reply:
Yep - I've also renamed the env var to "project" for more clarity